src/adagrams.js

```diff
@@ -1,8 +1,124 @@
 const Adagrams = {
   drawLetters() {
     // Implement this method for wave 1
+    let quantityByLetter = {
```

Comment:
This should be a `const`. Style note: move this object outside the function `drawLetters()` but inside the module `Adagrams`